libs/pika/async_cuda_base/include/pika/async_cuda_base/cuda_stream.hpp

```diff
@@ -49,9 +60,25 @@ namespace pika::cuda::experimental {
         PIKA_EXPORT cuda_stream(cuda_stream const&);
         PIKA_EXPORT cuda_stream& operator=(cuda_stream const&);
 
+        /// \brief Get the underlying stream.
+        ///
+        /// \return the underlying stream. The stream is still owned by the \ref cuda_stream and
+        /// must not be manually released.
         PIKA_EXPORT whip::stream_t get() const noexcept;
+
+        /// \brief Get the device of the stream.
+        ///
+        /// \return the device of the stream.
         PIKA_EXPORT int get_device() const noexcept;
+
+        /// \brief Get the priority of the stream.
+        ///
+        /// \return the priority of the stream.
```

Comment:
Should we just do:
```suggestion
        /// \brief Get the priority of the stream.
```
for simple functions like these?

Reply:
Simplify for getters and setters.